### ISSUE TITLE
English Translate

### ISSUE THREAD

**Issue by DemonRem** [hdnpt/geartrack#68]
can it's be in english :(

**Comment by carlosflorencio** [comment]
Hi,

Yes, the new version of the website will be in english.
https://geartrack.pt

Stay tuned.

**Comment by DemonRem** [comment]
it's will be here on the src too?

**Comment by carlosflorencio** [comment]
No, like the readme says there will be no more updates to this repo. 
The updates will reflect the online website only. 😞 

What is your use case?
